Makefile

```diff
@@ -163,6 +163,19 @@ kfserving-install: knative-install cert-manager-install
 seldon-install: istio-install
 	@./scripts/seldon-operator-install.sh
 
+new-k8s-cluster:
+	@./scripts/ci/k3d.sh
+
+mlflow-e2e:
+	@./scripts/ci/mlflow-e2e.sh
+
+fuseml-install:
+	@./dist/fuseml-installer version
+	@./dist/fuseml-installer install
+
+test-mlflow-e2e: build new-k8s-cluster kfserving-install fuseml-install mlflow-e2e
+
+
```

Comment:
What about cluster delete?

Reply:
I have added a target for deleting the cluster, however keep in mind that make is not intended to be a framework test so the `cleanup` function does not exist. In that way, running make without `-k` will not delete the cluster in case of failure.

Reply:
Good